context/app/static/js/components/Routes/useSendWebVitals.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import ReactGA from 'react-ga';
+import { getCLS, getFID, getLCP, getTTFB } from 'web-vitals';
+
+function sendVitalsToGA({ name, delta, id }) {
+  ReactGA.event({
+    category: 'Web Vitals',
+    action: name,
```

Comment:
```suggestion
    action: (name === 'CLS' ? 'CLS * 1000' : name),
```
If we ended up using a different multiplier, we'd want to distinguish new and old data?

Reply:
or maybe not: Like not to tweak copy-and-paste code without a good reason. (Also wondering is other metrics are worth multiplying.)